money-monkey-web/lib/db/index.ts

```
@@ -1,6 +1,7 @@
-import { drizzle } from "drizzle-orm/vercel-postgres";
-import { sql } from "@vercel/postgres";
+import { drizzle } from "drizzle-orm/postgres-js";
+import postgres from "postgres";
 
 import * as schema from "@/lib/db/schema";
 
-export const db = drizzle(sql, { schema });
+const client = postgres(process.env.DATABASE_URL!, { prepare: false });
```

Biome

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)